<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAirlineIcaoFilterBar</class>
 <widget class="QFrame" name="CAirlineIcaoFilterBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>24</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Airline ICAO filter</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gl_AirlineIcaoFilterBar">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>4</number>
   </property>
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <item row="0" column="2">
    <widget class="QLabel" name="lbl_AirlineId">
     <property name="text">
      <string>Id:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLineEdit" name="le_Designator">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="maxLength">
      <number>4</number>
     </property>
     <property name="placeholderText">
      <string>ICAO</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="lbl_Designator">
     <property name="text">
      <string>Airline:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="12">
    <spacer name="hs_FilterBar">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="7">
    <widget class="QLineEdit" name="le_Name">
     <property name="placeholderText">
      <string>Name of airline</string>
     </property>
    </widget>
   </item>
   <item row="0" column="10">
    <widget class="QCheckBox" name="cb_RealAirline">
     <property name="text">
      <string>Real airline</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="lbl_Name">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="11">
    <widget class="QCheckBox" name="cb_VirtualAirline">
     <property name="text">
      <string>Virtual airline</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QLabel" name="lbl_Country">
     <property name="text">
      <string>Country:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="swift::gui::filters::CFilterBarButtons" name="filter_Buttons"/>
   </item>
   <item row="0" column="9">
    <widget class="swift::gui::components::CDbCountrySelectorComponent" name="country_Selector">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="le_AirlineId">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>id</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::filters::CFilterBarButtons</class>
   <extends>QFrame</extends>
   <header>gui/filters/filterbarbuttons.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CDbCountrySelectorComponent</class>
   <extends>QFrame</extends>
   <header>gui/components/dbcountryselectorcomponent.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>le_Designator</tabstop>
  <tabstop>le_Name</tabstop>
  <tabstop>cb_RealAirline</tabstop>
  <tabstop>cb_VirtualAirline</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
